src/grandorgue/sound/GOSoundFilter.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2006 Milan Digital Audio LLC
+ * Copyright 2009-2024 GrandOrgue contributors (see AUTHORS)
+ * License GPL-2.0 or later
+ * (https://www.gnu.org/licenses/old-licenses/gpl-2.0.html).
+ */
+
+#ifndef GOSOUNDFILTER_H_
+#define GOSOUNDFILTER_H_
+
+#include <cmath>
+
+enum class GO_FILTER_TYPE { LPF, HPF, LOW_SHELF, HIGH_SHELF };
+
+class GOSoundFilter {
+public:
+  class FilterState {
+  public:
+    FilterState() { Init(nullptr); }
+    void Init(const GOSoundFilter *filter) {
```

Comment:
Init() is called rarerly than ProcessBuffer(), so the implementation is not needed to be inline and should be moved to .cpp.